src/main/java/com/discordsrv/placeholderapi/DSRVExpansion.java

```diff
@@ -97,13 +98,13 @@ public String onPlaceholderRequest(Player player, String identifier) {
             case "guild_bot_game_url":
                 return getOrEmptyString(mainGuild.getSelfMember(), member -> member.getActivities().stream().findFirst().map(Activity::getUrl).orElse(""));
             case "guild_members_online":
-                return String.valueOf(membersOnline.size());
+                return String.valueOf(membersOnline.get().size());
             case "guild_members_total":
                 return String.valueOf(mainGuild.getMembers().size());
             case "linked_online":
-                return String.valueOf(linkedAccounts.stream().filter(membersOnline::contains).count());
+                return String.valueOf(linkedAccounts.get().stream().filter(membersOnline.get()::contains).count());
```

Comment:
Running `Supplier#get` for `membersOnline` within the `Stream#filter` would cause it to get the supplier once for every linked account, this would be counter productive
```suggestion
                List<String> onlineMembers = membersOnline.get();
                return String.valueOf(linkedAccounts.get().stream().filter(onlineMembers::contains).count());
```